tools/bgpreader.c

```diff
@@ -446,6 +469,23 @@ int main(int argc, char *argv[])
       usage();
       goto done;
       break;
+#ifdef WITH_RPKI
+    case 500:
+      bgpstream_rpki_parse_ssh(optarg, rpki_input);
+      break;
+    case 501:
+      bgpstream_rpki_parse_collectors(optarg, rpki_input);
+      break;
+    case 502:
+      bgpstream_rpki_parse_live(rpki_input);
+      break;
+    case 503:
+      bgpstream_rpki_parse_unified(rpki_input);
+      break;
+    case 504:
+      bgpstream_rpki_parse_default(rpki_input);
+      break;
+#endif
```

Comment:
@salsh is the naming of these arguments normal? `-50x` is hard to understand without actually checking the source code.

Reply:
oh, i see. i guess you (@salsh)  wanted long args without short args, so the 50x are a way to ensure no normal argument collides with them.
i think this is fine, but let's create an enum at the top of bgpreader that gives these values a more friendly name.

Reply:
> oh, i see. i guess you (@salsh) wanted long args without short args, so the 50x are a way to ensure no normal argument collides with them.

Exactly.

> i think this is fine, but let's create an enum at the top of bgpreader that gives these values a more friendly name.

Alright, done!